projects/user_publication/semantic_scholar.py

```diff
@@ -131,8 +131,8 @@ def _get_pub_type(types, forum):
     return "other"
 
 
-def _save_publication(publication, dry_run=True):
-    title = publication.get("title")
+def _get_pub_model(publication, dry_run=True):
+    title = utils.decode_unicode_text(publication.get("title"))
```

Comment:
Will this require us to re-encode the text every time we display it?

Reply:
This is a good question. My understanding is we only need to decode before putting things into the database, as it's mysql with the unicode issues. It looks like this function is called with a publication that might be either from scopus/semantic scholar or also the database?

Reply:
Yes, this is only from scopus/semantic and not from the database. If we indent to show it in the website with same encoding then I think changing database to allow unicode chars is the way to go.